### PR TITLE
Propagate counter on tick

### PR DESCRIPTION
**Pull request by catalin-me**
This PR adds the necessary logic to propagate the `counter` on the `tick` event. The new adjustments are also covered by integration tests.

### CODE REVIEW

**Comment by catalin-me** [comment]
@mcollina Sure, I'll take care of it today. :)

**Comment by catalin-me** [comment]
@mcollina I just updated the `README` as well. Please let me know your thoughts.

File: lib/run.js [hunk 1]
```diff
@@ -134,9 +134,9 @@ function _run (opts, cb, tracker) {
     totalCompletedRequests += counter
     requests.recordValue(counter)
     throughput.recordValue(bytes)
+    tracker.emit('tick', counter)
```
Comment (thekemkid) [inline lib/run.js]: What are your thoughts on making this line like this? 
```suggestion
    tracker.emit('tick', {counter, bytes})
```

If you think users would want access to the `count` per tick, would it be fair to think they'd want `bytes` per tick, too?

Comment (catalin-me) [reply lib/run.js]: Great suggestion. I just updated the event. :)

**Pull request merged**